dora/services/migrations/0105_update_orientation_modes.py

```diff
@@ -0,0 +1,126 @@
+from django.conf import settings
+from django.db import migrations
+from django.db.models import Q
+
+from dora.services.enums import ServiceStatus
+
+
+def is_orientable(service):
+    structure_blacklisted = False
+    for siren in settings.ORIENTATION_SIRENE_BLACKLIST:
+        if service.structure.siret and service.structure.siret.startswith(siren):
+            structure_blacklisted = True
+            break
```

Comment:
if siret := service.structure.siret:
        structure_blacklisted = siret[0:9] in settings.ORIENTATION_SIRENE_BLACKLIST